### ISSUE TITLE
Add Quantity + Note fields into Service line (row) for Quotes + Invoices

### ISSUE THREAD

**Issue by Mausino** [salesagility/SuiteCRM#1793]
#### Expected Behavior

It will possible to add the quantity of service + note text
#### Actual Behavior

We can't add note + quantity into service line
#### Context

We solved our problem by adding the Quantity field to Service line (row) :) I am sure that will help to other users by everyday using of SuiteCRM

I advice to add this new field + also the field Noti field to "Service row" into quotes (also into invoices) in SuiteCRM. First i reported as bug #844 

![quantity](https://cloud.githubusercontent.com/assets/5773777/17168317/12fae544-53e2-11e6-9ff3-1d9ed952d3d9.png)

**Comment by samus-aran** [comment]
![](https://github.trello.services/images/mini-trello-icon.png) [Add Quantity + Note fields into Service line (row) for Quotes + Invoices](https://trello.com/c/F2whQ3rQ/82-add-quantity-note-fields-into-service-line-row-for-quotes-invoices)

**Comment by samus-aran** [comment]
We have now set up a new home for suggestions at Trello.  All github issues that were labeled 'suggestion' have been moved and will be closed.  Certain ones will be progressed within the new Suggestion Box and may be re-opened.

Announcement of moving Suggestions:
https://suitecrm.com/forum/suggestion-box/13691-moving-suggestions-from-github-to

New SuiteCRM Suggestion Box
https://trello.com/b/Ht7LbMqw/suitecrm-suggestion-box
